<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>threadsManagerClass</class>
 <widget class="QMainWindow" name="threadsManagerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>331</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>threadsManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>431</width>
      <height>151</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
    </property>
    <column>
     <property name="text">
      <string>TID</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Priority</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>CPU Time</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="suspendButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Suspend</string>
    </property>
   </widget>
   <widget class="QComboBox" name="priorityComboBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Idle</string>
    </property>
    <item>
     <property name="text">
      <string>Idle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Below Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Above Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Realtime</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="numOfThreadsComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string>1</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="runButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resumeButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="killButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Kill</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chooseSyncComboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Mutex</string>
    </property>
    <item>
     <property name="text">
      <string>Mutex</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Semaphore</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Waiting funtions</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No sync</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="refreshLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="reloadButton">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="threadsManager.qrc"/>
 </resources>
 <connections/>
</ui>
